src/controllers/api/imageController.js

```diff
@@ -0,0 +1,30 @@
+import aws from 'aws-sdk';
+import mime from 'mime-types';
+
+const s3 = new aws.S3({ params: { Bucket: 'dnd-5e-api-images' } });
+
+const show = async (req, res, next) => {
+  try {
+    const params = {
+      Bucket: 'dnd-5e-api-images',
+      Key: req.url.slice(1),
+      Range: req.headers.range,
+    };
+    return s3.getObject(params, function(err, data) {
+      if (err) {
+        res.status(200);
+        res.end('Error Fetching File');
+      } else {
+        res.writeHead(200, { 'Content-Type': mime.lookup(params.Key) });
```

Comment:
Shouldn't you be able to get the content type from `data.Headers.ContentType`? Instead of relying on the extension (which is what I assume mime.lookup does)

Reply:
That's an interesting point! I'll see if I can do that.

Reply:
It ended up being `data.ContentType` but good catch!